pages/linux/cmus.md

```diff
@@ -3,10 +3,14 @@
 > Commandline Music Player.
 > Use arrow keys to navigate, `<enter/return>` to select, and numbers 1-8 switch between different views.
 
-- Open cmus from specified directory:
+- Open cmus into specified directory (note that you will exit cmus to this directory):
 
 `cmus {{path/to/directory}}`
 
+- Open cmus into specified directory then return to original directory on exit:
+
+`cd {{path/to/directory}} && cmus && cd -`
```

Comment:
:thinking: this is not an example about the command itself, it's more a bash feature.

Reply:
this is true. if this doesn't fit within tldr, i understand completely.

my case for adding this example is that cmus doesn't seem to have this functionality natively and it was a pet peeve of mine when i was using cmus. to my mind this is how `cmus {{path/to/directory}}` should work, so having an example that not only gives that functionality but lets people know it's not the default functionality could be helpful.

i'd love to hear what you and others more experienced with tldr think @navarroaxel.

Reply:
I'd say this is something which would fit into the command description. For example:
> \> When opening `cmus` to a specific directory, it will change the working directory to that directory.

Reply:
that makes perfect sense. i can make the sort of change later today.

Reply:
sorry for the delay, but i just committed that change.